### PR TITLE
Revert back the  preparation done for v2.0.0  release

### PR DESCRIPTION
**Pull request by Madhu-1**
Revert back the  preparation done for v2.0.0  release

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

File: charts/ceph-csi-rbd/values.yaml [hunk 1]
```diff
@@ -104,7 +104,8 @@ nodeplugin:
   plugin:
     image:
       repository: quay.io/cephcsi/cephcsi
-      tag: 2.0.0
+      tag: v2.0-canary
+
```
Comment (nixpanic) [inline charts/ceph-csi-rbd/values.yaml]: Additional empty line?

**Comment by Madhu-1** [comment]
> In future, branch before making the change to master and the 1st commit in the branch would be the change of the version. Tag the commit in the branch for release.

@nixpanic we have not changed any images tags in the master, This PR is reverting back the image tag changes in `release-v2.0` branch. after doing the release we don't want to update the release image in quay when a PR is backported to release-v2.0 branch

File: charts/ceph-csi-rbd/values.yaml (hunk 1 above)
Comment (Madhu-1) [reply charts/ceph-csi-rbd/values.yaml]: removed

**Comment by nixpanic** [comment]
> @nixpanic we have not changed any images tags in the master, This PR is reverting back the image tag changes in release-v2.0 branch. after doing the release we don't want to update the release image in quay when a PR is backported to release-v2.0 branch

Ah, thanks! This is still a pretty ugly process. But I understand #610 has been reported for something similar already.

**Pull request merged**